content/1.numpy_intro.md

```diff
@@ -8,10 +8,14 @@
 
 :::
 
-:::{exercise} Time
-15 - 20 Minutes
+:::{instructor-note}
+  - Teaching : 10 min
+  - Demo: 10 min
+  - Discussions:0 min
```

Comment:
There is a discussion box, but it is rather a explanation by instructor. i.e. discussions should have time for learners to interact in the shared doc